docs/data/data-grid/column-menu/ColumnMenuGridReferences.js

```diff
@@ -0,0 +1,251 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DataGridPremium,
+  useGridApiRef,
+  useKeepGroupedColumnsHidden,
+} from '@mui/x-data-grid-premium';
+import Chip from '@mui/material/Chip';
+import Avatar from '@mui/material/Avatar';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+
+const getChipProperties = (plan) => {
+  switch (plan.toLowerCase()) {
+    case 'premium':
+      return { avatarLink: '/static/x/premium.svg', color: '#ffecc8' };
+    case 'pro':
+      return { avatarLink: '/static/x/pro.svg', color: '#c8e9ff' };
+    default:
+      return { avatarLink: undefined, color: '#c8ffdb' };
+  }
+};
+
+function PlanTag(props) {
+  if (!props.plan) {
+    return null;
+  }
+  const chipPropperties = getChipProperties(props.plan);
+  const avatar = !chipPropperties.avatarLink ? undefined : (
+    <Avatar src={chipPropperties.avatarLink} />
+  );
+
+  return (
+    <Stack sx={{ width: '100%' }}>
+      <Chip
+        avatar={avatar}
+        sx={{ background: chipPropperties.color, color: 'rgba(0, 0, 0, 0.87)' }}
+        label={props.plan}
+      />
+    </Stack>
+  );
+}
+
+PlanTag.propTypes = {
+  plan: PropTypes.string,
+};
+
+function ComponentTag(props) {
+  if (!props.value) {
+    return null;
+  }
+  const components = props.value.split(',');
+  return (
+    <Stack gap={0.5}>
+      {components.map((c, key) => (
+        <Typography
+          key={key}
+          sx={{
+            borderRadius: '5px',
+            background: 'rgba(102, 178, 255, 0.15)',
+            fontSize: '0.8rem',
+          }}
+        >
+          {c}
+        </Typography>
+      ))}
+    </Stack>
+  );
+}
+
+ComponentTag.propTypes = {
+  value: PropTypes.string,
+};
+
+const columns = [
+  {
+    field: 'slot',
+    headerName: 'Slot',
+    width: 220,
+    renderCell: (params) => <ComponentTag value={params.value} />,
+  },
+  {
+    field: 'plan',
+    headerName: 'Plan',
+    width: 170,
+    type: 'singleSelect',
+    valueOptions: ['Premium', 'Pro', 'Community'],
+    renderCell: (params) => <PlanTag plan={params.value} />,
+  },
+  { field: 'design', headerName: 'Menu Design' },
+  {
+    field: 'defaultComponent',
+    headerName: 'Default Components',
+    width: 280,
+    renderCell: (params) => <ComponentTag value={params.value} />,
+  },
+  { field: 'displayOrder', headerName: 'Display Order', width: 140, type: 'number' },
+];
+
+const rows = [
+  {
+    id: 1,
+    slot: 'ColumnMenuSortItem',
+    defaultComponent: '<GridColumnMenuSortItem />',
```

Comment:
```suggestion
    defaultComponent: 'GridColumnMenuSortItem',
```

Maybe remove the brackets to avoid confusion?
```tsx
ColumnMenuSortItem: GridColumnMenuSortItem // this works

ColumnMenuSortItem: <GridColumnMenuSortItem /> // this doesn't work
```